etc/clang-format-all.sh

```diff
@@ -21,11 +22,11 @@ source_dirs=(
 )
 
 mapfile -t source_files < <(
-  find "${source_dirs[@]:?}" -regextype posix-egrep -regex '.*\.(hpp|hh|cpp)'
+  find "${source_dirs[@]:?}" | grep -E '.*\.(hpp|hh|cpp)$'
```

Comment:
Although I would prefer to use PCRE (`-P`) for consistency, IIRC it is not fully supported across all distros/versions, so I am fine with using `-E` here for now.

Reply:
Unfortunately `-P` does not appear supported on macOS `grep`:

```
% grep -P
grep: invalid option -- P
usage: grep [-abcdDEFGHhIiJLlMmnOopqRSsUVvwXxZz] [-A num] [-B num] [-C[num]]
        [-e pattern] [-f file] [--binary-files=value] [--color=when]
        [--context[=num]] [--directories=action] [--label] [--line-buffered]
        [--null] [pattern] [file ...]
```